flink-streaming-java/src/test/java/org/apache/flink/streaming/runtime/tasks/StreamTaskTerminationTest.java

```diff
@@ -227,21 +229,13 @@ protected void cleanup() throws Exception {
 			// has been stopped
 			CLEANUP_LATCH.trigger();
 
-			// wait until handle async exception has been called to proceed with the termination of the
-			// StreamTask
-			HANDLE_ASYNC_EXCEPTION_LATCH.await();
```

Comment:
This handle should be deleted since it is no longer needed.

Reply:
Already done in a later commit.